src/pages/Login/Login.jsx

```diff
@@ -0,0 +1,74 @@
+import NavBar from "../../components/NavBar/NavBar"
+import "./login.css"
+import { useToken } from "../../hooks/useToken"
+import axios from "axios"
+import {  useNavigate } from "react-router-dom"
+const Login = ()=>{
+  const {setToken}=useToken();
+  const testCredentials= { email: "adarshbalika@gmail.com",password: "adarshbalika"};
+  let navigate= useNavigate();
+  const loginHandler=async()=>{
+    try{
+      const response = await axios.post('/api/auth/login', {...testCredentials});
+      if(response.status===200)
```

Comment:
I think when we are doing post method it's status code 201. Or you can give both here 200 and 201

Reply:
ya actually I was getting status as 200, I'll provide both